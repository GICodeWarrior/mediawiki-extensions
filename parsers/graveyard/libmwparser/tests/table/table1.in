{|
| (Cell 1, row 1)
| (Cell 2, row 1)
|-
| (Cell 1, row 2)
| (Cell 2, row 2)
|}
