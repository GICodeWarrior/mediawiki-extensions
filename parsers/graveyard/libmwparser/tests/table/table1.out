BEGIN ARTICLE
    BEGIN TABLE
        BEGIN TABLE BODY
            BEGIN ROW
                BEGIN CELL
                    SPACE[ ]
                    SPECIAL[(]
                    WORD[Cell]
                    SPACE[ ]
                    WORD[1]
                    SPECIAL[,]
                    SPACE[ ]
                    WORD[row]
                    SPACE[ ]
                    WORD[1]
                    SPECIAL[)]
                    NEWLINE
                END CELL
                BEGIN CELL
                    SPACE[ ]
                    SPECIAL[(]
                    WORD[Cell]
                    SPACE[ ]
                    WORD[2]
                    SPECIAL[,]
                    SPACE[ ]
                    WORD[row]
                    SPACE[ ]
                    WORD[1]
                    SPECIAL[)]
                    NEWLINE
                END CELL
            END ROW
            BEGIN ROW
                BEGIN CELL
                    SPACE[ ]
                    SPECIAL[(]
                    WORD[Cell]
                    SPACE[ ]
                    WORD[1]
                    SPECIAL[,]
                    SPACE[ ]
                    WORD[row]
                    SPACE[ ]
                    WORD[2]
                    SPECIAL[)]
                    NEWLINE
                END CELL
                BEGIN CELL
                    SPACE[ ]
                    SPECIAL[(]
                    WORD[Cell]
                    SPACE[ ]
                    WORD[2]
                    SPECIAL[,]
                    SPACE[ ]
                    WORD[row]
                    SPACE[ ]
                    WORD[2]
                    SPECIAL[)]
                    NEWLINE
                END CELL
            END ROW
        END TABLE BODY
    END TABLE
END ARTICLE
